content/en/xtc-release-notes/v43.md

```diff
@@ -0,0 +1,29 @@
+---
+title: XTC v43
+linkTitle: XTC v43
+
+weight: 15
```

Comment:
Ordering for release notes should be latest first (which you did), but as we do not know how many versions we'll still need in the future (see e.g. Firefox, currently at ninety-something) you should start at a pretty high weight (e.g. 200) and go downward from there (by 1 for each version - we *do* know that there won't be new versions popping up in between).